### ISSUE TITLE
add more images

### ISSUE THREAD

**Issue by skippy** [sjifire/website#53]
edited: this is mostly covered in #54; once we figure that out, I can go through and add images, whether in carousel, featured img, within the posts...

**Comment by skippy** [comment]
carousel on the home page?  on the about page?

**Comment by skippy** [comment]
@icegulch I'm going to assign to you to see what you think about having rotationg imgs in a few places (homepage header, homepage footer, who-we-are footer, etc). and if it should be a carousel or some other functionality.

since this has evolved, and we are getting more imgs up on every page as we work through them, moving this to v2

**Comment by icegulch** [comment]
doable, for sure.

i'd want to convert the current hero image from a css background-image to an image in html. that would make admin'ing far, far easier, while allowing us to fade-between or randomly pick images more easily. if we wanted footer images, this approach would allow us to lazily load that image when a user nears the footer (as opposed to on page load).

so, for the rotation, i could see the following options:

1. running fade-between (css)
2. choose a random image on page load (js)
3. choose a random image on build (gh action or push)

does this sound like what you were envisioning?

**Comment by skippy** [comment]
per discussion, this is shifting to focusing on the carousel widget that we can add to the home page, who-we-are, and perhaps a few other places... but starting with home page for now.

**Comment by skippy** [comment]
@kvenegassjifire I bumped the # of blog posts back up to 8 because @icegulch fixed the issue on mobile and on the desktop, 8 is the # that works well with the sidebar (setting to 4 leaves a huge gap at the bottom of the site).

However, if we leave it at 4, we can put a carousel/gallery of images in that location.  @kvenegassjifire , does that work for you?  

@icegulch thoughts on implementation?  I think the [cloudinary gallery](https://cloudinary.com/documentation/product_gallery) looks fine, esp. since all our images are hosted there already

**Comment by icegulch** [comment]
@skippy I like it because it's drop-dead simple! any reason to think through how to exclude a given image? happy to punt on that if there's no need for it now...i'm sure you could introduce that as a later feature!

**Comment by icegulch** [comment]
hey @skippy! i just tried a quick implementation of the Cloudinary gallery widget on the homepage. The one thing we're missing that I think we need is to have images "tagged"; the gallery searches by "tag", in other words. I no longer have Cloudinary access, so I can get in to see what tags might be there just to test. However, I'm pretty sure that we can set the fetch or upload parameters to auto-tag images from our domain. Regardless, this should be a pretty quick implementation.

**Comment by skippy** [comment]
@icegulch the challenge is we don't expose cloudinary at all to the admin users... I think we need to have the images be available in the admin tool.  So If the cloudinary widget doesn't buy us time, then I would say lets just use whatever.  If the cloudinary can take a list of images, that's great.

**Comment by icegulch** [comment]
@skippy to be sure I'm on the same page: are you saying it'd be prudent to let folks select which images appear in the gallery? that would make sense, but I just wanted to be sure. if we were okay with _any uploaded image_ being part of the gallery, we should be able to do that with cloudinary. if we wanted to allow users to select images, i still think we could do that with a fetch / upload preset in cloudinary. if either of these sounds appealing, let me know. if i'm utterly mis-understanding, also let me know! 😄

**Comment by skippy** [comment]
We’re on the same page!  Let’s make sure the admin can control the imgs. I
can geek out and try various options later like a folder of imgs or
something, but let’s make sure we start with control via the admin page.

Thanks sir!

**Comment by icegulch** [comment]
Hi again, @skippy! I just pushed up a new branch (`iss53`). It's not "plumbed" but I created a simple responsive gallery similar to Cloudinary's plugin. The idea would be to show 5 photos, but we could likely do more with a scrollable thumbnail navigation. Just wanted to showcase the concept with some placeholder pix. Let me know what you think.
![ 2023-01-13 at 09 23 47](https://user-images.githubusercontent.com/794708/212369274-66c2ef7b-cba8-4db3-a4e2-bfa14919a2c2.png)
